### PR TITLE
Catalog changelog 03 07 2024

### PR DESCRIPTION
**Pull request by SkylineKevinMOR**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -9,6 +9,97 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+- **PATCH api/user-catalog/v2-0/catalogs/{catalogId}/ranges/{rangeId}/state?organizationId={organizationId}**
+
+  Set the state of a specific range of a specific catalog item.
+- **PATCH api/user-catalog/v2-0/catalogs/{catalogId}/versions/{versionId}/state?organizationId={organizationId}**
+
+  Set the state of a specific version of a specific catalog item.
+
+#### 3 July 2024 - New Feature - Catalog - Change state support [ID_40096]
+
+When viewing an item on the catalog, it will now be possible to change the state of a range or version.
```
[the remark this answers is not in the document]
Comment (SkylineKevinMOR) [reply release-notes/Cloud/DCP_change_log.md:72]: Currently only Skyline members are allowed to change the state.

File: release-notes/Cloud/DCP_change_log.md [hunk 2]
```diff
@@ -9,6 +9,97 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+#### 3 July 2024 - New Feature - Catalog - Custom tag support [ID_40030]
+
+When viewing an item on the catalog, it will now be possible to give a custom tag to ranges.
+
+#### 3 July 2024 - New Feature - Catalog - Version history [ID_39903]
+
+When viewing an item on the catalog, the versions tab will now include a version history section.
+The version history section will by default show all supported ranges and versions.
+A toggle has also been introduced to include unsupported ranges and versions.
```
[the remark this answers is not in the document]
Comment (SkylineKevinMOR) [reply release-notes/Cloud/DCP_change_log.md:90]: Indeed the list is what you can expect toggling the toggle button

File: release-notes/Cloud/DCP_change_log.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md:72]:
```suggestion
Skyline employees will now be able to change the state of a range or version when they view an item in the Catalog.
```

File: release-notes/Cloud/DCP_change_log.md [hunk 3]
```diff
@@ -9,6 +9,97 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+  Set the state of a specific version of a specific catalog item.
+
+#### 3 July 2024 - New Feature - Catalog - Change state support [ID_40096]
+
+When viewing an item on the catalog, it will now be possible to change the state of a range or version.
+
+#### 3 July 2024 - New Feature - Catalog - Custom tag support [ID_40030]
+
+When viewing an item on the catalog, it will now be possible to give a custom tag to ranges.
```
[the remark this answers is not in the document]
Comment (SkylineKevinMOR) [reply release-notes/Cloud/DCP_change_log.md:80]: Only users that have access to that organization will be able to update/delete custom tags on ranges

[the remark this answers is not in the document]
Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md:80]: What do you mean with "that organization"? Are the tags linked to organizations?

File: release-notes/Cloud/DCP_change_log.md [hunk 4]
```diff
@@ -9,6 +9,97 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 3 July 2024 - New Feature - Catalog - Version 2 of the User Catalog API
```
[the remark this answers is not in the document]
Comment (SkylineKevinMOR) [reply release-notes/Cloud/DCP_change_log.md:12]: The API should be exposed to the public so would probably be beneficial to have it on there aswell.

File: release-notes/Cloud/DCP_change_log.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SkylineKevinMOR) [reply release-notes/Cloud/DCP_change_log.md:80]: Everyone is able to view the tags, but only the users that are part of the organization that published the item can update/delete tags

File: release-notes/Cloud/DCP_change_log.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md:12]: OK, I'll look into adding the API in the user guide. Another question: does this RN have an intranet ID, or is there no intranet entry for this one?

**Comment by SkylineKevinMOR** [comment]
Added the RNs for the api change, should be ready to merge now

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**